assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentPeerAssessmentServiceImpl.java

```diff
@@ -56,10 +63,11 @@
 @Slf4j
 public class AssignmentPeerAssessmentServiceImpl extends HibernateDaoSupport implements AssignmentPeerAssessmentService {
 
-    private ScheduledInvocationManager scheduledInvocationManager;
-    protected AssignmentService assignmentService;
-    private SecurityService securityService = null;
-    private SessionManager sessionManager;
+    @Getter @Setter private ScheduledInvocationManager scheduledInvocationManager;
+    @Getter @Setter protected AssignmentService assignmentService;
+    @Getter @Setter private SecurityService securityService = null;
+    @Getter @Setter private SessionManager sessionManager;
+    @Getter @Setter private SiteService siteService;
```

Comment:
most of the time you only need Setter for services

Reply:
Yes, I think we can remove the getters, right @bgarciaentornos?

Reply:
Hey Markus @stetsche, I think we can remove them, however, I'm not a big fan of changing code that is working and has been verified, even if they are simple and small changes. Let's take advantage of the effort spent on this, this is running in production for many years right now.

I prefer to not introduce differences between Master and the 22.x version of the feature, we did that in the past and that practices almost killed all of our rubrics features, this has been around for five years and we want to be consistent between versions.

Let's stick with what it is in 22.x please.